open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/nvd/CpeMatch.java

```diff
@@ -27,6 +28,7 @@
  * CPE match string or range
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
CPE match does not declare `additionalProperties: false` in schema, so retained.

Reply:
Retained as the schema allows additional properties: https://github.com/jeremylong/Open-Vulnerability-Project/blob/83189a18225029b1cb535f747e6b7662acfd0ecd/open-vulnerability-clients/src/main/resources/json/cve_api_json_2.0.schema#L419-L451